ui/selectors/institutional/selectors.ts

```diff
@@ -172,12 +172,13 @@ export function getIsCustodianSupportedChain(
     // @ts-expect-error state types don't match
     const selectedAccount = getSelectedInternalAccount(state);
     const accountType = getAccountType(state);
-    const providerConfig = getProviderConfig(state);
 
-    if (!selectedAccount || !accountType || !providerConfig) {
+    if (!selectedAccount || !accountType) {
       throw new Error('Invalid state');
     }
 
+    const providerConfig = getProviderConfig(state);
+
```

Comment:
One of two logic changes.

`getProviderConfig` now throws on its own if there isn't a `providerConfig`. This is teeccchnniiicaallly not supposed to be possible. See the implementation and changes to understand more.